src/handlers.js

```diff
@@ -339,21 +354,23 @@ async function jobHandler(message) {
           '```\n',
           '</details>',
         ].join('\n');
-        await instGithub.repos.createCommitComment({
+        //console.log(instGithub.pullRequests);
+        //console.log('..pr number' organization);
+        await instGithub.pullRequests.createComment({
```

Comment:
We only want to use `createComment` instead of `createCommitComment` if this is a pull request.

Reply:
This  (and the next call) is enclosed by the `if (message.payload.details['event.type'].startsWith('pull_request.'))` check on L341 above , so these are executed only in case of pull requests